docs/CodeDesign.md

```diff
@@ -1,20 +1,23 @@
 # Requirements
 
-## Prototype
+## Initial goals
 
-The goal of the first prototype is to reach a working and usable tool with minimal functionality.
-Namely, a user would be able to run `fawltydeps` as a command line tool in their Python project.
-FawltyDeps will:
+We want a working and usable tool that slowly grows functionality as needed.
+A user would be able to run `fawltydeps` as a command line tool in their Python
+project, and immediately get some useful and actionable information.
 
-- Automatically discover all Python code files and extract all the packages imported in the code.
-- Automatically discover and extract from files declaring dependencies (supporting requirements.txt, setup.py and pyproject.toml).
-- Use identity mapping between extracted imports and declared dependencies,
-- Compare the two sets (of used vs declared dependencies) and report missing and unused dependencies.
-- Support Python version 3.7 - 3.11
+## Boundaries
+
+- Support all current Python versions, that means all Python versions that have
+  a stable release, and are not yet End Of Life. Currently this is: v3.7 - v3.11.
+- For now we support the CPython interpreter only
+- OS-wise, for now we have concentrated on Linux. We should still run fine on
```

Comment:
Repetition of "for now" that is a bit jarring

Reply:
```suggestion
- For now we support the CPython interpreter only
- OS-wise, we have concentrated on Linux. We should still run fine on
```